.gitignore

```diff
@@ -5,4 +5,5 @@
 *.iml
 *.ipr
 *.iws
-/podman/production.conf
\ No newline at end of file
+/podman/production.conf
+/tatoonie_extracted/
```

Comment:
What is this new directory doing?

Reply:
Sorry this was by mistake, it was folder where build part was extracted. I have reverted back .gitignore file. Please ignore this!